src/pudl/output/pudltabl.py

```diff
@@ -802,14 +807,40 @@ def gen_original_eia923(self, update=False):
             )
         return self._dfs["gen_og_eia923"]
 
+    def gen_fuel_allocated_generator_fuel_type_eia923(self, update=False):
```

Comment:
There are so many of these gen/net gen/gen fuel functions now I feel like they need a naming convention, or at least a standard way of describing what the primary key columns are, and where the data columns came from. It still feels like a mix of ad-hoc names. Like I think this one has a PK of `report_date`, `plant_id_eia`, `generator_id`, `prime_mover_code` and `fuel_type_code` (or is it `energy_source_code` now?) but prime mover doesn't show up in the function name, while it is included in `gen_pm_fuel_ownership` below.